cms/src/components/content/ContentOverview.vue

```diff
@@ -131,6 +171,104 @@ router.currentRoute.value.meta.title = `${capitaliseFirstLetter(titleType)} over
+                                <input
+                                    class="ml-2 text-zinc-800 focus:border-0 focus:outline-none focus:ring-0"
+                                    type="radio"
+                                    name="sortOption"
+                                    value="title"
+                                    v-model="selectedSortOption"
+                                    data-test="sort-option-title"
+                                />
+                                Title
+                            </label>
+                            <label>
+                                <input
+                                    class="ml-2 text-zinc-800 focus:border-0 focus:outline-none focus:ring-0"
+                                    type="radio"
+                                    name="sortOption"
+                                    value="expiryDate"
+                                    v-model="selectedSortOption"
+                                    data-test="sort-option-expiry-date"
+                                />
+                                Expiry Date
+                            </label>
+                            <label>
+                                <input
+                                    class="ml-2 text-zinc-800 focus:border-0 focus:outline-none focus:ring-0"
+                                    type="radio"
+                                    name="sortOption"
+                                    value="publishDate"
+                                    v-model="selectedSortOption"
+                                    data-test="sort-option-publish-date"
+                                />
+                                Publish Date
+                            </label>
+                            <label>
+                                <input
+                                    class="ml-2 text-zinc-800 focus:border-0 focus:outline-none focus:ring-0"
+                                    type="radio"
+                                    name="sortOption"
+                                    value="updatedTimeUtc"
+                                    v-model="selectedSortOption"
+                                    data-test="sort-option-last-updated"
+                                />
+                                Last Updated
+                            </label>
+                        </div>
+                        <hr class="my-2" />
+                        <div class="flex flex-col gap-1">
+                            <button
+                                @click="handleSortAscending"
+                                class="flex w-full gap-1 rounded-md p-2 text-zinc-950 hover:bg-zinc-300"
+                                :style="{
+                                    backgroundColor: sortingAscending ? '#18181b' : 'white',
+                                    color: sortingAscending ? 'white' : '#18181b',
+                                }"
```

Comment:
I have the same comment here as on the other PR. Since this doesn't necessarily make sense as an `LButton` (it's a situational button with different styling), I would in this case just style it with a dynamic `:class`. We could take that up with the rest of the styling.